app/src/renderer/vuex/modules/governance/proposals.js

```diff
@@ -52,12 +58,13 @@ export default ({ node }) => {
         Raven.captureException(error)
         state.error = error
       }
-      state.loading = false
     },
     async getProposal({ state, commit }, proposal_id) {
       state.loading = true
       try {
         state.error = null
+        state.loading = false
+        state.loaded = true // TODO make state for only proposal
```

Comment:
can you clarify this `TODO`?

Reply:
As we are doing a request here, we should probably have `loading`, `loaded` states for it. I wasn't sure if we want another module or a substate `proposals.proposal.[loading, loaded, error]` here. Ideas?

Reply:
ohhhh i see. good question. not sure what the vue-recommended way of doing this is.